package.json

```diff
@@ -1,31 +1,26 @@
 {
-  "name": "@prosperworks/eslint-config-prosperworks",
-  "version": "1.1.2",
+  "name": "@prosperworks/eslint-config",
+  "version": "2.0.0",
   "description": "A mostly reasonable approach to JavaScript.",
-  "scripts": {
-    "prelint": "editorconfig-tools check ./rules/*",
-    "lint": "eslint ./rules",
-    "fix": "eslint ./rules --fix"
-  },
-  "repository": "git+https://git@github.com/ProsperWorks/js-style.git",
-  "homepage": "https://github.com/ProsperWorks/js-style",
```

Comment:
do we want to get rid of this homepage link? i'm totally speaking without context, but i would think that maybe npm would parse this file to figure out how to link the page and the repo.

or is it that by doing `github:ProsperWorks/js-style` you get the same benefits?

Reply:
the repo is linked via `repository`, see an example npm page https://www.npmjs.com/package/lodash

homepage would be for a marketing landing or doc site or something